src/util/graph.h

```diff
@@ -255,11 +255,6 @@ class grapht
   void disconnect_unreachable(node_indext src);
   void disconnect_unreachable(const std::vector<node_indext> &src);
 
-  std::vector<node_indext> depth_limited_search(
-    const node_indext &src,
-    std::size_t &limit,
-    bool forwards) const;
-
```

Comment:
Why is this a mistake?  Is there a body that needs removal as well?

Reply:
No, the mistake is that the body was already removed

Reply:
OK.  In future it would be nice to have that in a commit message.

Reply:
It would be good to amend the commit message according to @martin-cs' comment.